components/header/index.css

```diff
@@ -8,9 +8,66 @@
   flex-basis: 60%;
   align-self: stretch;
   height: inherit;
+  width: 100%;
+  transition: transform 0.3s ease-in, border 0s ease-in 0.3s;
+  will-change: transform;
```

Comment:
I do not really get where you do change transform.

Reply:
It's here. https://github.com/oacore/dashboard/blob/mobile-view/components/header/index.css#L56 However the animation doesn't work properly. I will need more time to make it look better.

Reply:
It's more about structure. I will create better design for you